app/controllers/usage_controller.rb

```diff
@@ -1,14 +1,26 @@
-# frozen_string_literal: true
```

Comment:
Please be careful to not remove _magic comments_, these serve a purpose

https://freelancing-gods.com/2017/07/27/friendly-frozen-string-literals.html

Rubocop will fail when these are removed.